src/test/java/de/qabel/desktop/cellValueFactory/BoxObjectCellValueFactoryTest.java

```diff
@@ -122,7 +122,8 @@ public void testDate() {
 
         column = getColumnValue("mtime", rootNodeFile);
         BoxFile subnode = (BoxFile) rootNodeFile.getChildren().get(0).getValue();
-        DateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+        //DateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+        DateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
Same here. You do the trivial check that `new SimpleDateFormat("MM/dd/yyyy").format(mtime)` is the same as `new SimpleDateFormat("MM/dd/yyyy").format(mtime)` which is obviously true because we know that `subnode == bf`. It would be better to expect a certain output here like `assertThat(column.getValue(), is(equalTo("01/02/2016")));`. That way the test doesn't have to care about the exact implementation but only about the expected outcome